packages/vtk/samples/convert-vtk/convert-vtk.ipynb

```diff
@@ -0,0 +1,103 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "765eb40e",
+   "metadata": {},
+   "source": [
+    "# Publish Geoscience objects from VTK files\n",
+    "\n",
+    "This notebook shows how you can login to Evo and publish Geoscience objects from VTK files to your chosen workspace.\n",
+    "\n",
+    "In this first cell we create a ServiceManagerWidget which will open a browser window and ask you to login.\n",
+    "\n",
+    "Once logged in, a widget will be displayed below allowing you to select an organisation and workspace to publish objects to.\n",
+    "\n",
+    "__Required:__ You must fill in your `client_id` before running this cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d88e33f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evo.notebooks import ServiceManagerWidget\n",
+    "\n",
+    "manager = await ServiceManagerWidget.with_auth_code(\n",
+    "    client_id=\"your-client-id\",\n",
+    "    oidc_issuer=\"https://ims.bentley.com\",\n",
+    "    discovery_url=\"https://discover.api.seequent.com\",\n",
+    "    redirect_url=\"http://localhost:3000/signin-oidc\",\n",
+    ").login()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f64b7d2",
+   "metadata": {},
+   "source": [
+    "In the cell below we choose the VTK file we want to publish and set its path in the `vtk_file` variable.\n",
+    "\n",
+    "Then we choose an EPSG code to use for the Coordinate Reference System.\n",
+    "\n",
+    "You may also specify tags to add to the created Geoscience objects.\n",
+    "\n",
+    "Then we call `convert_vtk`, passing it the VTK file path, EPSG code, the service manager widget from above, and finally a path we want the published objects to appear under.\n",
+    "\n",
+    "Then we loop over the results from the `convert_vtk` function, printing out each object that was published to Evo.\n",
+    "\n",
+    "_Note:_ Some geometry types are not yet supported. A warning will be shown for each element that could not be converted."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b7e33971",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pprint\n",
+    "\n",
+    "from evo.data_converters.vtk.importer import convert_vtk\n",
+    "\n",
+    "vtk_file = os.path.join(os.getcwd(), \"data/input/collection.vtm\")\n",
+    "epsg_code = 32650\n",
+    "\n",
+    "tags = {\"Source\": \"Jupyter Notebook\"}\n",
+    "\n",
+    "objects_metadata = convert_vtk(\n",
+    "    filepath=vtk_file, epsg_code=epsg_code, service_manager_widget=manager, tags=tags, upload_path=\"notebook\"\n",
+    ")\n",
+    "\n",
+    "print(\"These objects have now been published:\")\n",
+    "\n",
+    "for metadata in objects_metadata:\n",
+    "    pprint.pp(metadata, indent=4)"
```

Comment:
Could you simply put `objects_metadata` instead of this loop and let Jupyter inspect the variable rather than pretty printing it?